lib/logstash/inputs/kafka.rb

```diff
@@ -354,4 +357,16 @@ def set_sasl_config(props)
 
     props.put("sasl.kerberos.service.name",sasl_kerberos_service_name) unless sasl_kerberos_service_name.nil?
   end
+
+  def timestamp_ms
+    (Time.now.to_f * 1000).to_i
+  end
+
+  def has_to_commit?(last_commit_time)
+    # If auto_commit is enable we just leave the commit to the client library on poll and close actions
+    return false if @enable_auto_commit == "false"
```

Comment:
this should be `return false if @enable_auto_commit == "true"` (we don't want to commit manually if auto commit is on)